src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+
+      <fieldset>
+        <legend class="form-field">
+          Personal information
+        </legend>
+  
+        <label class="form-field">
+          Surname:
+          <input type="text" name="Surname">
+        </label>
+  
+        <label class="form-field">
+          Name:
+          <input type="name">
```

Comment:
There is no type `name `
<img width="430" alt="image" src="https://user-images.githubusercontent.com/99403198/214551533-4b492372-9d53-4426-aa42-792ebcf8f154.png">